pyopia/statistics.py

```diff
@@ -424,6 +425,25 @@ def make_montage(stats_file, pixel_size, roidir,
     return montage
 
 
+def montage_plot(montage, pixel_size):
```

Comment:
this should be in the 'future' plotting module. I will do a PR to move this shortly.

Reply:
see #79